src/signals/incident-management/reducer.js

```diff
@@ -149,11 +149,13 @@ export default (state = initialState, action) => {
         .set('activeFilter', initialState.get('activeFilter'))
         .set('editFilter', initialState.get('editFilter'))
         .set('ordering', initialState.get('ordering'))
+        .set('loading', true)
         .set('page', initialState.get('page'))
         .set('searchQuery', action.payload);
 
     case RESET_SEARCH_QUERY:
       return state
+        .set('loading', true)
```

Comment:
Should this set `loading` to `true`?

Reply:
Yeah, it should, because both setting and resetting the search query will trigger an incident fetch. 

Ideally, `loading` should be set to 'true' when the `searchIncidents` action is dispatched, but that will result in an infinite loop, because of the way the saga routes and triggers requests.